src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -67,7 +65,7 @@
         public static final int INTAKE_1 = 0, INTAKE_2 = 0, INTAKE_UP = 0, INTAKE_DOWN = 0;
 
         // index can ids are range 40-49
-        public static final int INDEX = 0;
+        public static final int INDEX_1 = 40, INDEX_2 = 41, INDEX_SENSOR_1 = 42, INDEX_SENSOR_2 = 43;
```

Comment:
Consider renaming index 1 and 2 constants to match the subsystem so there is no confusion.